101-bundle-json.md

```diff
@@ -500,4 +506,45 @@ The built-in actions (`install`, `upgrade`, `uninstall`) MUST NOT appear in the
 
 Implementations that do not support custom actions MUST NOT emit errors (either runtime or validation) if a bundle defines custom actions. That is, even if an implementation cannot execute custom actions, it MUST NOT fail to operate on bundles that declare custom actions.
 
+## Custom Extensions
+
+In many cases, the bundle descriptor is a sufficient artifact for delivering a CNAB bundle, since invocation images and other images may be retrieved from registries and repositories. Consequently, it is important to provide an extension mechanism. A _custom extension_ is a named collection of auxiliary data whose meaning is defined outside of this specification.
```

Comment:
Maybe "However" would be more appropriate than "Consequently" unless the previous sentence gave an example which _isn't_ covered by just the standard bundle descriptor data.

Reply:
Fixed